packages/flutter/lib/src/rendering/image.dart

```diff
@@ -182,6 +193,22 @@ class RenderImage extends RenderBox {
     markNeedsPaint();
   }
 
+  /// Whether the image should be flipped horizontally.
+  ///
+  /// This is occasionally used with images in right-to-left environments, for
```

Comment:
I'd put something in here with `[TextDirection]` so that folks reading the docs can get back to the "main article".

Reply:
done here and in RawImage